ansible/roles/fastpath/tasks/main.yml

```diff
@@ -83,15 +95,15 @@
   template:
     src: templates/fastpath.conf
     dest: "/opt/{{fastpath_user}}/backend/fastpath/fastpath.conf"
-    mode: 0444
+    mode: "0744"
```

Comment:
Why is this being changed to `rwx`? I think we had this pattern of making them read-only so that we enforce that editing the config should always go through ansible.

Reply:
It was due a permission issue that I was dealing with docker. But I'll try to rework it again because the are similar issues in other places